packages/devtools_app/lib/src/logging/flutter/logging_screen.dart

```diff
@@ -98,6 +113,113 @@ class LogsTable extends StatelessWidget {
       columns: columns,
       data: data,
       keyFactory: (LogData data) => ValueKey<LogData>(data),
+      onItemSelected: onItemSelected,
+    );
+  }
+}
+
+class LogDetails extends StatefulWidget {
+  const LogDetails({Key key, @required this.log}) : super(key: key);
+  final LogData log;
+
+  @override
+  _LogDetailsState createState() => _LogDetailsState();
+}
+
+class _LogDetailsState extends State<LogDetails>
+    with SingleTickerProviderStateMixin {
+  AnimationController crossFade;
+
+  @override
+  void initState() {
+    super.initState();
+    crossFade = AnimationController(
+      vsync: this,
+      duration: const Duration(milliseconds: 200),
```

Comment:
we should soon do an audit of all durations used and add some top level duration constants used throughout the app.